core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changelog.xml

```diff
@@ -132,4 +132,5 @@
     <include file="changesets/20241121_billing_account_changes_status.xml" relativeToChangelogFile="true"/>
     <include file="changesets/20241218_alter_audit_workflow_status_trigger_frequency.xml" relativeToChangelogFile="true"/>
     <include file="changesets/20250211_add_workspace_spend_report_table.xml" relativeToChangelogFile="true"/>
+    <include file="changesets/20250227_update_workspace_spend_report_table.xml" relativeToChangelogFile="true"/>
```

Comment:
this file name does not match which is causing the test failures

Reply:
Ah copy/paste error 🤦‍♀️ Thanks!